packages-native/animation/src/Animation.xml

```diff
@@ -0,0 +1,200 @@
+                        <enumerationValue key="bounceOutRight">Bounce right</enumerationValue>
+                        <enumerationValue key="fadeOut">Fade</enumerationValue>
+                        <enumerationValue key="fadeOutDown">Fade down</enumerationValue>
+                        <enumerationValue key="fadeOutDownBig">Fade down big</enumerationValue>
+                        <enumerationValue key="fadeOutUp">Fade up</enumerationValue>
+                        <enumerationValue key="fadeOutUpBig">Fade up big</enumerationValue>
+                        <enumerationValue key="fadeOutLeft">Fade left</enumerationValue>
+                        <enumerationValue key="fadeOutLeftBig">Fade left big</enumerationValue>
+                        <enumerationValue key="fadeOutRight">Fade right</enumerationValue>
+                        <enumerationValue key="fadeOutRightBig">Fade right big</enumerationValue>
+                        <enumerationValue key="flipOutX">Flip X</enumerationValue>
+                        <enumerationValue key="flipOutY">Flip Y</enumerationValue>
+                        <enumerationValue key="lightSpeedOut">Light speed</enumerationValue>
+                        <enumerationValue key="slideOutDown">Slide down</enumerationValue>
+                        <enumerationValue key="slideOutUp">Slide up</enumerationValue>
+                        <enumerationValue key="slideOutLeft">Slide left</enumerationValue>
+                        <enumerationValue key="slideOutRight">Slide right</enumerationValue>
+                        <enumerationValue key="zoomOut">Zoom</enumerationValue>
+                        <enumerationValue key="zoomOutDown">Zoom down</enumerationValue>
+                        <enumerationValue key="zoomOutUp">Zoom up</enumerationValue>
+                        <enumerationValue key="zoomOutLeft">Zoom left</enumerationValue>
+                        <enumerationValue key="zoomOutRight">Zoom right</enumerationValue>
+                    </enumerationValues>
+                </property>
+            </propertyGroup>
+
+            <propertyGroup caption="Timing">
+                <property key="duration" type="integer" defaultValue="200">
+                    <caption>Duration (ms)</caption>
+                    <description/>
+                </property>
+                <property key="delay" type="integer" defaultValue="0">
+                    <caption>Delay (ms)</caption>
+                    <description/>
+                </property>
+            </propertyGroup>
+
+            <propertyGroup caption="Condition">
+                <systemProperty key="Visibility"/>
+                <property key="condition" type="expression" required="false">
+                    <caption>Expression</caption>
+                    <description>The animation will start after it becomes visible while also obeying its timing delay. For already-visible content, an expression can be used to start the animation.</description>
```

Comment:
Just to clarify, property "condition" is not responsible for visibility of the component but to start the animation ?

Reply:
Yes, total condition for start; visible, after delay and expression is true (or empty)